### PR TITLE
Update ops doc with cloud-platform-notify channel info

### PR DESCRIPTION
**Pull request by sablumiah**
Update ops doc with cloud-platform-notify channel info

### CODE REVIEW

File: docs/INFO-our-ops-processes.md [hunk 1]
```diff
@@ -61,7 +62,25 @@ When the solution is a quick one, it is good to have the whole conversation in t
...
 
-Sometimes it is useful to make broadcast communications, examples include:
+### User support tickets
+
+If someone is asking for help that will be quick to do (less than 15 mins) or is mainly advice then keep the interaction in channel and get it done.
+
+If someone needs something that takes longer, is more challenging to complete or you find you've spent longer than 15 mins on it, then continue to talk in the channel but also ask the person asking for help to raise a ticket using the GitHub issue link: [long version](https://github.com/ministryofjustice/cloud-platform/issues/new?template=cloud-platform-support-request.md&labels=support%20team) and [short version](https://goo.gl/msfGiS).
+
+The purpose of the ticket is to keep a record of the work we are doing and how it is progressing. It is *not* the primary communication channel with the person who raised the problem &mdash; this should remain slack where you can provide a richer and more human interaction, answering questions if necessary. The engineer working on the ticket should update the ticket for our record as work progresses.
```
Comment (alkar) [inline docs/INFO-our-ops-processes.md]: Tiny thing, can you convert `&mdash;` to an ascii dash please?

File: docs/INFO-our-ops-processes.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ This is a record of the operational processes that we will use to support our us
 
 ## Hours of support
 
-Our hours of providing support are 10am - 5pm. During this time we will work on support requests from teams and make sure someone is available to answer questions in [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/).
+Our hours of providing support are 10am - 5pm. During this time we will work on support requests from teams and make sure someone is available to answer questions in [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) and look at PR request notifications for the cloud-platform-environments repo on [`#cloud-platform-update`](https://mojdt.slack.com/messages/CH6D099DF/)
```
Comment (digitalronin) [inline docs/INFO-our-ops-processes.md]: #cloud-platform-update is the broadcast channel we use to announce stuff. I think you mean #cloud-platform-notify for automated notifications (like PRs)

File: docs/INFO-our-ops-processes.md [hunk 3]
```diff
@@ -61,7 +62,25 @@ When the solution is a quick one, it is good to have the whole conversation in t
...
+
+In many cases it can be more helpful for the engineer on support to create the ticket rather than the person reporting the problem as you will be able to provide more relevant context.
+
+Once they have created the ticket it will appear in the `Support To Do` column of our [sprint board](https://waffle.io/ministryofjustice/cloud-platform). It can then be moved into the relevant columns as you work on it.
+
+**Note**
+
+New support tickets added by users who are not part of the WebOps Github team will not automatically appear in the Support To Do column yet. This is due to Github permissions not being fine-grained enough to permit MoJ organisation users who are not on WebOps to create the necessary label. This should be fixed soon. Until it is you may see some support tickets appear in Inbox and not Support To Do.
+
+## `#cloud-platform-notify` slack channel
```
Comment (digitalronin) [inline docs/INFO-our-ops-processes.md]: Do you mean #cloud-platform-update here?

File: docs/INFO-our-ops-processes.md (hunk 1 above)
Comment (sablumiah) [reply docs/INFO-our-ops-processes.md]: Done

File: docs/INFO-our-ops-processes.md (hunk 2 above)
Comment (sablumiah) [reply docs/INFO-our-ops-processes.md]: Corrected

File: docs/INFO-our-ops-processes.md (hunk 3 above)
Comment (sablumiah) [reply docs/INFO-our-ops-processes.md]: Corrected

**Pull request merged**